app/controllers/user_sessions_controller.rb

```diff
@@ -143,7 +143,7 @@ def handle_site_login_flow
                   session[:return_to] = nil
                   redirect_to return_to + hash_params
                 elsif params[:return_to]
-                  redirect_to params[:return_to] + hash_params
```

Comment:
Update:
 By removing `hash_params` the routing is correct now. It is now moving to /subscriptions but tags are not subscribed. Checking how to fix it.

Reply:
I receive flash[:notice] 'Successfully logged in.' now